doc/api/permissions.md

```diff
@@ -550,6 +550,10 @@ There are constraints you need to know before using this system:
 * Relative paths are not supported through the CLI (`--allow-fs-*`).
 * The model does not inherit to a child node process.
 * The model does not inherit to a worker thread.
+* Symbolic links will be followed even to locations outside of the set of paths
```

Comment:
I think that's not the correct assumption to state here. Symbolic links are not followed. We just don't handle symbolic links at all. I think it's clear to say: Symbolic links are handled as regular file paths.

Reply:
How about

Relative symbolic links within the allowed paths may allow an attacker to access files and directories which were not allowed. When starting applications with permissions enabled you much ensure that any files or directories to which access is allowed are not relative symbolic links and do not contain relative symbolic links.

Reply:
Michael's version is clearly documenting the boundary and responsibility scopes.